backend/api/Areas/Documents/DocumentController.cs

```diff
@@ -185,6 +185,20 @@ public IActionResult GetDocumentStorageTypes()
             return new JsonResult(result);
         }
 
+        /// <summary>
+        /// Retrieves the list of document types.
+        /// </summary>
+        [HttpGet("storage/types/{mayanDocumentTypeId}/metadata")]
+
+        // [HasPermission(Permissions.PropertyAdd)]
```

Comment:
Please add the correct permissions now that they are availiable

Reply:
done